index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main class="container" id="main">
+      <!-- Header section-->
+      <header>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </header>
+      <!-- Video Section -->
+      <div class="section" id="img-div">
+        <img src="./images/tribute.jpg" id="image" alt="" />
```

Comment:
Ensure you add the alt description to your image tag to ensure a better search engine ranking